### ISSUE TITLE
Unable to set SCHED_FIFO scheduling 

### ISSUE THREAD

**Issue by yuqin11** [scylladb/scylla#6089]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 2.3.1
Cluster size:1
OS (RHEL/CentOS/Ubuntu/AWS AMI):centos 7.2

*Hardware details (for performance issues)*          Delete if unneeded
Platform (physical/VM/cloud instance type/docker):docker
Hardware: sockets= cores= hyperthreading= memory= smp=4 mem=8G
Disks: (SSD/HDD, count) SSD xfs

```
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: 
WARNING: unable to mbind shard memory; performance may suffer: WARNING: unable to mbind shard memory; performance may suffer: WARNING: unable to mbind shard memory; performance may suffer: 
WARN  2020-03-27 08:25:01,065 [shard 0] seastar - Unable to set SCHED_FIFO scheduling policy for timer thread; latency impact possible. Try adding CAP_SYS_NICE
WARN  2020-03-27 08:25:01,662 [shard 0] iotune - nomerges for /sys/devices/pci0000:00/0000:00:05.0/virtio2/block/vdb/queue/nomerges set to 0. It is recommend to set it to 2 before evaluation so that merges are disabled. Results can be skewed otherwise.
2020-03-27 08:26:56,528 WARN Included extra file "/etc/supervisord.conf.d/scylla-housekeeping.conf" during parsing
2020-03-27 08:26:56,528 WARN Included extra file "/etc/supervisord.conf.d/scylla-jmx.conf" during parsing
2020-03-27 08:26:56,528 WARN Included extra file "/etc/supervisord.conf.d/scylla-server.conf" during parsing
WARN  2020-03-27 08:26:57,796 [shard 0] seastar - Unable to set SCHED_FIFO scheduling policy for timer thread; latency impact possible. Try adding CAP_SYS_NICE
WARN  2020-03-27 08:26:57,853 [shard 0] init - Using default cluster name is not recommended. Using a unique cluster name will reduce the chance of adding nodes to the wrong cluster by mistake
WARN  2020-03-27 08:26:57,913 [shard 0] database - Skipping undefined keyspace: view_pending_updates
WARN  2020-03-27 08:39:42,878 [shard 0] init - I/O Scheduler is not properly configured! This is a non-supported setup, and performance is expected to be unpredictably bad.
 Reason found: none of --max-io-requests, --io-properties and --io-properties-file are set.
To properly configure the I/O Scheduler, run the scylla_io_setup utility shipped with Scylla.
```

**Comment by yuqin11** [comment]
does these warnings lead to bad performance?

**Comment by slivne** [comment]
@penberg can you please check this out and try and help

**Comment by slivne** [comment]
seems to be related to #6017 - closing this till we fix #6017
